wallet/src/actors/app/handlers/create_vtt.rs

```diff
@@ -40,6 +41,8 @@ pub struct CreateVttRequest {
     session_id: types::SessionId,
     wallet_id: String,
     fee_type: Option<FeeType>,
+    #[serde(default)]
+    utxo_strategy: UtxoSelectionStrategy,
```

Comment:
```suggestion
    utxo_strategy: Option<UtxoSelectionStrategy>,
```

I might make sense to have the UTXO selection strategy as an optional field to be backwards compatible with older Sheikah versions. AFAIK, Sheikah tries to download latest wallet version even it is an older version.

Reply:
Now I see the `serde(default)`. i will test it! :) Thanks

Reply:
`serde(default)` will use the default value if it's missing (Random), so it should be backwards compatible.